app/views/users/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<div id="clearance" class="sign-up">
+  <h2><%= t(".title") %></h2>
+  <%= form_for @user do |form| %>
+    <%= render partial: "/users/form", object: form %>
```

Comment:
If I change the partial to ```render "form"``` it cannot find the path to the form. You can see from the screen shot it looks in
`/otaku/app/views`
`gems/clearance-1.12.0/app/views`

<img width="1425" alt="screen shot 2015-12-07 at 5 00 25 pm" src="https://cloud.githubusercontent.com/assets/529465/11643967/fc70cc7e-9d04-11e5-805b-e36d96b16fff.png">